rstudio/rstudio
# use :focus-visible pseudoclass instead of focus-visible polyfill

### Intent

Addresses:
- [remove focus-visible polyfill #14352](https://github.com/rstudio/rstudio/issues/14352)

### Approach

Use standard `:focus-visible` pseudoclass instead of the `focus-visible` polyfill (all browsers now support the pseudoclass and the polyfill was causing issues with the Visual Editor in recent Chrome releases).

Eliminated the "Always show focus outlines" accessibility option. These are now always on. If we decide we still want this option it could be done via some additional css work and applying a class to the page body based on the preference.

### Automated Tests

None

### QA Notes

Test that tabbing between controls with the keyboard shows focus rectangles, but clicking on controls does not. This is all controlled by the browser. The focus rectangle styles should be the same as before (for better or worse), so you could compare against an earlier build to see if it's consistent.

### Documentation

None

### Checklist
